modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,60 @@
+<7> `parent`: (Optional) Specifies the parent menu item under which the current item is nested. If this field is used, the parent menu item must be defined elsewhere in the `menuItems` configuration of any enabled plugin. You can define this field for each section.
+
+. To add *Main menu item* to the dynamic plugins configuration file, add the `default.` prefix to its key in your `{my-app-config-file}` file:
 +
-For more complex, multi-segment paths such as `path: /metrics/users/info`, the `menu_item_name` must use dot notation to represent the full path, for example, `metrics.users.info`. Trailing and leading slashes in paths are ignored. For example, `path: /docs` results in `menu_item_name: docs`, and `path: /metrics/users` results in `menu_item_name: metrics.users`.
-
-<2> This optional attribute specifies the icon for the menu item. You can use default icons or extend the icon set with dynamic plugins. {product-short} also provides additional icons in its internal library, such as: 
-+
-.Home Icon in the internal library
-[source, yaml]
-----
-dynamicPlugins:
-  frontend:
-    <package_name>:
-      menuItems:
-        <menu_item_name>: 
-          icon: home
-----
-+
-Similarly, the internal library includes icons for `group`, `category`, `extension`, and `school`. If the icon is already defined in the `dynamicRoutes` configuration under `menuItem.icon`, it can be removed from the in the `menuItems` configuration. Also, both SVG and HTML image icons are supported. For example:
-+
-.Example SVG icon
-[source,html]
-----
-icon: <svg width="20px" height="20px" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" fill="#ffffff">...</svg>
-----
-+
-.Example image icon
-[source,html]
-----
-icon: https://img.icons8.com/ios-glyphs/20/FFFFFF/shop.png
-----
-
-<3> This optional attribute specifies the title of the menu item. It can be removed if the title is already specified in the `dynamicRoutes` configuration under `menuItem.text`.
-
-<4> This optional attribute sets the order in which menu items appear in the sidebar. The default priority is 0, which places the item at the bottom of the list. A higher priority value places the item higher in the sidebar. You can define this attribute for each section.
-
-<5> This optional attribute specifies the parent menu item under which the current item is nested. If this attribute is used, the parent menu item must be defined elsewhere in the `menuItems` configuration of any enabled plugin. You can define this attribute for each section.
-
-.Example `menuItems` configuration
 [source,yaml]
 ----
 dynamicPlugins:
   frontend:
-    <package_name>:
-      dynamicRoutes:
-        - path: /my-plugin
-          module: CustomModule
-          importName: FooPluginPage
-          menuItem:
-            icon: fooIcon
-            text: Foo Plugin Page
+    default.main-menu-items: # <1>
```

Comment:
Omit the callout. There is nothing to edit here for the user, therefore it is more logical to move the description in the step general text.

```suggestion
    default.main-menu-items:
```